flybywiresim/aircraft
# feat(fms): allow SID name to be displayed on the ND

Fixes #[issue_no]

## Summary of Changes

Allows FMS to publish SID identifier to ND as found on A380 & A350 aircraft currently. 
## Screenshots (if necessary)


![image](https://github.com/user-attachments/assets/b5f54656-2afa-4c64-bc50-0039384f6a36)

## References

A380/A350 FCOM, YT videos & preview picture of feature on A320



## Additional context



Discord username (if different from GitHub):
bruno_pt99
## Testing instructions

A32NX: Perform a normal flight and verify approach identifier shows on the ND as before, in addition, no SID name should be displayed on the ND during preflight & takeoff/climb (like the attached picture). 
A380:
1) Insert a SID on the flightplan and verify the SID name appears on top of ND.
2) Fly the SID and perform a direct to a point of the SID prior to the final one, SID name should remain visible.
3) Verify SID name disappears after last waypoint of SID is sequenced, or, after performing a direct to the last SID waypoint.
4) Insert a SID and cruise flight level in a way that it will be reached before the end of SID. Verify that the SID name disappears once cruise level is reached.

## How to download the PR for QA

Every new commit to this PR will cause new A32NX and A380X artifacts to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, find and click on the **PR Build** tab
1. Click on either **flybywire-aircraft-a320-neo** or **flybywire-aircraft-a380-842** download link at the bottom of the page
